website/docs/usage/101/_named-entities.md

```diff
@@ -15,19 +21,20 @@ nlp = spacy.load("en_core_web_sm")
 doc = nlp(u"Apple is looking at buying U.K. startup for $1 billion")
 
 for ent in doc.ents:
-    print(ent.text, ent.start_char, ent.end_char, ent.label_)
+    print(ent.text, ent.start_char, ent.end_char, ent.label_, ent.kb_id_)
```

Comment:
I'm not sure we should include this here, because it's not something that users will get out-of-the-box. We likely won't be shipping the entity linking with the core models by default, so having this in the 101 is potentially confusing.

Reply:
Agreed! (I wasn't sure whether it would be shipped as part of the models or not)